src/site/_includes/partials/post.njk

```diff
@@ -116,7 +116,8 @@
     {% if codelabs %}
       {% CodelabsCallout codelabs, lang %}
     {% endif %}
-    <web-feedback additional-questions="{{ feedback }}"></web-feedback>
+    {# TODO: Consider removing the feedback widget as noisy and unhelpful. #}
```

Comment:
This is phrased with the wrong tense. We have removed it now - maybe something like:

{# TODO: We've removed the feedback widget for now as it's noisy and unhelpful. #}